frontend/src/pages/projects/screens/detail/ProjectDetails.tsx

```diff
@@ -27,6 +29,16 @@ const ProjectDetails: React.FC = () => {
   const modelServingEnabled = featureFlagEnabled(
     dashboardConfig.spec.dashboardConfig.disableModelServing,
   );
+  const queryParams = useQueryParams();
+  const notebookLogout = queryParams.get('notebook_logout');
+  const notification = useNotification();
+
+  // TODO: this will be triggered twice, don't know why
+  React.useEffect(() => {
+    if (notebookLogout) {
+      notification.success(`Logout workbench successfully`);
+    }
+  }, [notebookLogout, notification]);
```

Comment:
make a custom hook -- this is all one thing and shouldn't be so inline with a component. Get into the habit of making custom hooks when you have isolated & coupled functionality.